core/src/main/java/org/radargun/stats/Statistics.java

```diff
@@ -37,14 +38,14 @@
    /**
     * @return Merged operation stats for registered groups. Calculation should be based on operations registered with this group.
     */
-   Map<String, OperationStats> getOperationStatsForGroups();
+   List<Map<String, OperationStats>> getOperationStatsForGroups();
```

Comment:
It should be rather Map<String, List, but the presence of this is wrong by design.